### ISSUE TITLE
LinkedinShareButton not working

### ISSUE THREAD

**Issue by shandysulen** [nygardk/react-share#261]
Sharing via `<LinkedinShareButton />` is not working.

```
<LinkedinShareButton url={shareLink}>
    <LinkedinIcon size={32} round={true} />
</LinkedinShareButton>
```
I imagine this is due to some update to LinkedIn's sharing API, so the `<LinkedinShareButton />` needs to be updated as a result.

**Comment by cretto** [comment]
Up

**Comment by amyiee** [comment]
Hey there, I'm wondering if you can re-visit this issue. it doesn't seem to work on my end

**Comment by WilliamChou06** [comment]
Looks like this is happening because LinkedIn cannot retrieve any metadata from the link to be shared.

You can reproduce this by changing the LinkedIn share button to point to any URL that generates a rich link (or not).

**Comment by nygardk** [comment]
Fixed in `v4.0.0`.
